### PR TITLE
CLI: add scan command

### PR DESCRIPTION
**Pull request by vancexu**

### CODE REVIEW

File: tools/cli/workflow.go [hunk 1]
```diff
@@ -148,6 +148,16 @@ func newWorkflowCommands() []cli.Command {
 				ListAllWorkflow(c)
 			},
 		},
+		{
+			Name:    "scan",
+			Aliases: []string{"sc", "scanall"},
+			Usage: "scan workflow executions (only work with ElasticSearch). " +
```
Comment (longquanzheng) [inline tools/cli/workflow.go]: nit: only works if Cadence server enabled visibility with ElasticSearch

File: tools/cli/flags.go [hunk 2]
```diff
@@ -406,6 +406,43 @@ func getFlagsForListAll() []cli.Flag {
 	}
 }
 
+func getFlagsForScan() []cli.Flag {
+	return []cli.Flag{
+		cli.IntFlag{
+			Name:  FlagPageSizeWithAlias,
```
Comment (longquanzheng) [inline tools/cli/flags.go]: you can set the default value here so that the --help will show the default value

File: tools/cli/flags.go [hunk 3]
```diff
@@ -406,6 +406,43 @@ func getFlagsForListAll() []cli.Flag {
...
+			Usage: "For tune page size",
```
Comment (longquanzheng) [inline tools/cli/flags.go]: maybe "page size of result"

File: tools/cli/app.go [hunk 4]
```diff
@@ -27,7 +27,7 @@ import (
 const (
 	// Version is the controlled version string. It should be updated every time
 	// before we release a new version.
-	Version = "0.8.1"
+	Version = "0.8.2"
```
Comment (yycptt) [inline tools/cli/app.go]: I think we have 0.8.3.

Comment (vancexu) [reply tools/cli/app.go]: Only for CLI. Compatible with server 0.8.*

**Pull request merged**